crates/bevy_pbr/src/render/prepass.rs

```diff
@@ -240,14 +234,14 @@ impl<M: Material> SpecializedMeshPipeline for DepthPrepassPipeline<M> {
 
         let vertex_buffer_layout = layout.get_layout(&vertex_attributes)?;
 
-        let fragment = if key.contains(MeshPipelineKey::DEPTH_PREPASS_NORMALS)
+        let fragment = if key.contains(MeshPipelineKey::PREPASS_NORMALS)
             || key.contains(MeshPipelineKey::ALPHA_MASK)
         {
             let frag_shader_handle = if let Some(handle) = &self.material_fragment_shader {
                 handle.clone()
             } else {
-                info!("no frag");
-                DEPTH_PREPASS_SHADER_HANDLE.typed::<Shader>()
+                warn!("Missing Material::prepass_fragment_shader() for material with UUID {}. Rendering may be incorrect.", M::TYPE_UUID);
```

Comment:
Oh, my old info log was just for debugging stuff. I'm not sure it's actually a mistake to not specify a custom prepass shader. Or at least, I'm not sure it deserves a warn, people just writing some simple materials shouldn't need to think about this I think. Maybe just log it as a trace instead?

Reply:
Yeah I'm not sure about this either. The shader/material stuff isn't clear to me, so I don't know if this is actually ever going to be a problem. We discussed issues with custom materials overwriting fragment/vertex shaders, and how that plays into writing normals, and how that interacts with alpha masks, but yeah. I'm not sure.